README.md

```diff
@@ -34,14 +34,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+Adaptive layout uses specific breakpoints in design to accomadate for specific devices and uses hard coded pixels. Responsive layout is for thousands of different devices and uses responsive units instead of hard coding in pixels.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+A desktop first layout is designed with the desktop viewport in mind and is then later scaled for other devices. A mobile first design is first implemented for mobile viewports and then later scaled to accomadate larger devices.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+rem is a relative length unit. Making font-size 62.5% in the html scales 1rem to 10px.
+
 4. How would you describe preprocessing to someone new to CSS?
 
+Preprocessing is a more powerful way of using CSS that allows you to condense lines of code by intruducing variables and functions (by way of mixins). Instead of repeatedly typing out the same styles, you can call variables and mixins instead.
+
 5. What is your favorite concept in preprocessing? What is the concept that gives you the most trouble?
 
+I really like the ability to set themes via variables and also being able to call mix-ins. They provide a way to create changes across the board by only creating a few changes of actual code.
+
+I'm still trying to strike the balance for a useful number of mix-ins - I feel like maybe I don't utilize it as often as I could, and also, conversely, maybe I create often useless ones as well.
+
```

Comment:
Feeling like you don't utilize them is completely normal due to the small scope of projects we do in Lambda, but in larger projects, this will rarely be an issue.